_includes/configuration/forced-root-block.md

```diff
@@ -1,14 +1,14 @@
-## forced_root_block
+## `forced_root_block`
 
 This option enables you to make sure that any non block elements or text nodes are wrapped in block elements. For example `<strong>something</strong>` will result in output like: `<p><strong>something</strong></p>`. This option is enabled by default as of version 3.0.
 
 If you set this option to `false` it will never produce `p` tags on enter, or, automatically it will instead produce `br` elements and `Shift+Enter` will produce a `p`.
 
-> Note: Not using `p` elements as root block can severely cripple the functionality of the editor.
+> **Warning**: Not using `p` elements as root block negatively affect the functionality of the editor.
```

Comment:
This doesn't quite read right to me though it could just be me. Would `affects` be more suitable?

Reply:
```suggestion
> **Warning**: Not using `p` elements as the root block will impair the functionality of the editor.
```

Reply:
I would be happy with that.